go.mod

```diff
@@ -7,59 +7,59 @@ require (
+	k8s.io/kubectl v0.20.6
+	k8s.io/kubernetes v1.19.10
 )
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.3.2
+	github.com/containerd/containerd => github.com/containerd/containerd v1.5.0
 	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20171107005402-dcf79f8c7fcb
 	github.com/godbus/dbus => github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f
 	// Needed to resolve broken transitive dependencies in buildkit and s2i
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200323165209-0ec3e9974c59
 	// these are needed since k8s.io/kubernetes cites v0.0.0 for these in its go.mod
-	k8s.io/api => k8s.io/api v0.20.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver => k8s.io/apiserver v0.20.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.0
-	k8s.io/client-go => k8s.io/client-go v0.20.0
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.0
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.0
-	k8s.io/code-generator => k8s.io/code-generator v0.20.0
-	k8s.io/component-base => k8s.io/component-base v0.20.0
-	k8s.io/cri-api => k8s.io/cri-api v0.20.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.0
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.0
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.0
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.0
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.0
-	k8s.io/kubectl => k8s.io/kubectl v0.20.0
-	k8s.io/kubelet => k8s.io/kubelet v0.20.0
-	k8s.io/kubernetes => k8s.io/kubernetes v1.19.3
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.0
-	k8s.io/metrics => k8s.io/metrics v0.20.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.0
+	k8s.io/api => k8s.io/api v0.20.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.20.6
+	k8s.io/apiserver => k8s.io/apiserver v0.20.6
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.6
+	k8s.io/client-go => k8s.io/client-go v0.20.6
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.6
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.6
+	k8s.io/code-generator => k8s.io/code-generator v0.20.6
+	k8s.io/component-base => k8s.io/component-base v0.20.6
+	k8s.io/cri-api => k8s.io/cri-api v0.20.6
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.6
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.6
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.6
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.6
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.6
+	k8s.io/kubectl => k8s.io/kubectl v0.20.6
+	k8s.io/kubelet => k8s.io/kubelet v0.20.6
+	k8s.io/kubernetes => k8s.io/kubernetes v1.19.10
```

Comment:
I don't think we need a replace here (I _think_). Bumping k8s.io/kubernetes to v1.21.0 above should be sufficient.

Reply:
Actually I wanted to keep it at `0.20.6`, but something keeps automatically bumping it up to `1.21` on `go mod vendor`. I'll investigate.